src/snowflake/cli/plugins/snowpark/common.py

```diff
@@ -15,23 +15,21 @@
 from __future__ import annotations
 
 import re
-from typing import Dict, List, Optional, Set, Union
+from typing import Dict, List, Optional, Set
 
 from snowflake.cli.api.constants import ObjectType
 from snowflake.cli.api.identifiers import FQN
-from snowflake.cli.api.project.schemas.snowpark.callable import (
-    FunctionSchema,
-    ProcedureSchema,
-)
+from snowflake.cli.api.project.schemas.entities.snowpark_entity import SnowparkEntity
 from snowflake.cli.api.sql_execution import SqlExecutionMixin
 from snowflake.cli.plugins.snowpark.models import Requirement
 from snowflake.cli.plugins.snowpark.package_utils import (
     generate_deploy_stage_name,
 )
 from snowflake.connector.cursor import SnowflakeCursor
 
-DEFAULT_RUNTIME = "3.8"
-FunctionOrProcedure = Union[FunctionSchema, ProcedureSchema]
+DEFAULT_RUNTIME = (
+    "3.8"  # TODO: Should we change this, as we plan to drop support for 3.8?
```

Comment:
Let's discuss this and potentially create another PR - could you create a jira ticket? I think BCR is a good occasion

Reply:
SNOW-952504

Reply:
SNOW-1571687 😄 
Thanks!